storage/backward_compatible_s3.go

```diff
@@ -140,7 +140,12 @@ func (s *BackwardCompatibleS3Storage) Put(key string, imageFile io.ReadSeeker, c
 		putMetadata[k] = aws.String(v)
 	}
 
-	_, err := s.client.PutObject(&s3.PutObjectInput{
+	_, err := imageFile.Seek(0, 0)
```

Comment:
ref https://github.com/aws/aws-sdk-go/issues/1962#issuecomment-406783345

As far as I tried on local environment, this code certainly fixed the upload problem.

I'm not sure, however, that the code should be placed here. 🙌